### ISSUE TITLE
Support for a CUDA 12.5.x 

### ISSUE THREAD

**Issue by jasonsmithio** [pytorch/serve#3278]
### 🚀 The feature

I pushed the container and saw over 500 CVEs when I scanned the container. Many, if not all, would be addressed if we support Ubuntu 24.04. NVIDA has a docker image running 24.04 as it's base and it runs 12.5.1. I would like to update the build scripts to support this imae

### Motivation, pitch

500+ CVEs related to the base image using 20.04

**Comment by agunapal** [comment]
Hi @jasonsmithio You can build your custom image using this flag

https://github.com/pytorch/serve/blob/master/docker/build_image.sh#L29

Last time we tried, we didn't have a good experience with the ubuntu22.04 runners for CI/CD and hence didn't upgrade to 22.04. Will try again.

**Comment by thenicekat** [comment]
@agunapal what exactly was the issue with upgrading it to 22.04 or 24.04? I am also building these images locally and wanted to know if it causes any issues with the actual working itself.

**Comment by thenicekat** [comment]
Fair enough on the CI/CD part, but does this imply the images will work as expected or does that require further testing?
